### ISSUE TITLE
FEATURE: parametric 'filter' regex

### ISSUE THREAD

**Issue by bitbay** [bezoerb/postcss-image-inliner#207]
The filter regex should be parametric to allow extension and new arrivals, e.g.:
```
imageInliner({
    assetPaths: [],     // List of directories where the inliner should look for assets
    maxFileSize: 10240  // Sets a max file size (in bytes)
    strict: false, // Fail on error
    filter: /^(background(?:-image)?)|(content)|(cursor)|(border-image)/  // defaults to /^(background(?:-image)?)|(content)|(cursor)/
})
```

**Comment by anotherjames** [comment]
`(list-style(?:-)image)?)` could be supported this way too.

**Comment by robinscholz** [comment]
This should be included for things like `mask` and `mask-image`

**Comment by bezoerb** [comment]
Sorry for the delay...
Option is available in [v6.2.0](https://github.com/bezoerb/postcss-image-inliner/releases/tag/v6.2.0)
